pkg/domain/infra/tunnel/containers.go

```diff
@@ -138,3 +139,24 @@ func (ic *ContainerEngine) ContainerRm(ctx context.Context, namesOrIds []string,
 	}
 	return reports, nil
 }
+
+func (ic *ContainerEngine) ContainerTop(ctx context.Context, options entities.TopOptions) (*entities.StringSliceReport, error) {
+	if options.Latest {
```

Comment:
there is a helper func for this if you like .... ctrs, err := getContainersByContext(ic.ClientCxt, options.All, namesOrIds)